aspnetcore/security/cors.md

```diff
@@ -13,6 +13,8 @@ uid: security/cors
 [!INCLUDE [not-latest-version](~/includes/not-latest-version.md)]
 :::moniker-end
 
+[!INCLUDE[](~/security/cors/includes/cors7.md)]
+
 :::moniker range=">= aspnetcore-8.0"
```

Comment:
@tdykstra  I forgot to put the default version first so the anchors keep working for the default version.